roles/smartgateway/templates/deployment.yaml.j2

```diff
@@ -23,15 +23,34 @@ spec:
         smart-gateway: '{{ meta.name }}'
     spec:
       containers:
+{% if service_type == 'metrics' %}
+      - name: bridge
+        image: '{{ bridge_container_image_path }}'
+        args:
+        - --amqp_url
+        - amqp://{{ amqp_url | default('messaging-internal-' + meta.name + '.' + meta.namespace + '.svc:5672/collectd/telemetry') }}
```

Comment:
Sure, I'll take a look at removing the hard-coded scheme once I finish a blocking bug or two with the STO. A couple of notes about this:
1) The existing amqp_url variable is not a URL without a scheme, so I'd like to rename that if I'm messing with this argument
2) This `default()` expression is being repeated around the codebase. I'd like to more sanely set the default URLs based on _scheme_ _host_ _port_ and _path_ variables. if we're going to address part of the problem, might as well do it all.
3) This is the only supported scheme at the moment, amqps is not supported, though it would be nice if it were. This connection is internal to the OCP cluster, so we could co-locate pods on the same host with affinity settings in order to further improve the security posture of this cleartext connection.